Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -177,6 +177,75 @@ class PlaygroundController: ObservableObject {
         return configuration
     }
 
+    var embeddedConfiguration: EmbeddedPaymentElement.Configuration {
+        var configuration = EmbeddedPaymentElement.Configuration(formSheetAction: .confirm(completion: { [weak self] result in
+            // TODO(porter) Handle two step confirm
```

Comment:
ticket or immediate followup? We should probably make an embedded-only `formSheetAction` toggle in the playground

Reply:
I can make a ticket, in my head I was thinking the week of Oct. 7 it would be added but I can add sooner

Reply:
I'll make a follow up right now and add the hide mandate API too.